uefi/flash.go

```diff
@@ -35,6 +37,57 @@ type FlashDescriptor struct {
 	ExtractPath string
 }
 
+func findSignature(buf []byte) (int, error) {
+	if bytes.Equal(buf[16:16+len(FlashSignature)], FlashSignature) {
+		// 16 + 4 since the descriptor starts after the signature
+		return 20, nil
+	}
+	if bytes.Equal(buf[:len(FlashSignature)], FlashSignature) {
+		// + 4 since the descriptor starts after the signature
+		return 4, nil
```

Comment:
is it worth defining a nice named constant?

Reply:
I'll do it.